modules/virt-creating-vm-cloned-pvc-data-volume-template.adoc

```diff
@@ -1,73 +1,75 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/creating_vms_custom/virt-creating-vms-by-cloning-pvcs.adoc
+// * virt/creating_vms_advanced/creating_vms_cli/virt-creating-vms-by-cloning-pvcs.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-creating-vm-cloning-pvc-data-volume-template_{context}"]
 = Creating a VM from a cloned PVC by using a data volume template
 
-You can create a virtual machine (VM) that clones the persistent volume claim (PVC) of an existing VM by using a data volume template.
-
-This method creates a data volume whose lifecycle is dependent on the original VM. Deleting the original VM deletes the cloned data volume and its associated PVC.
+You can create a virtual machine (VM) that clones the persistent volume claim (PVC) of an existing VM by using a data volume template. This method creates a data volume whose lifecycle is independent on the original VM.
 
 .Prerequisites
 
 * The VM with the source PVC must be powered down.
+* You must have the `virtctl` command line tool installed.
```

Comment:
```suggestion
* You must have the `virtctl` command-line tool installed.
```

Reply:
Thank you. For the record, "comman-line" is also my preference, but in the openshift-docs repository, it seems to be a coin toss and what ultimately convinced me to leave a space was how it is introduced in the OpenShift web console.

```
$ grep -hroe 'command[ -]line tool' | sort | uniq -c
     22 command line tool
     23 command-line tool
```

Reply:
That is a bummer that there are so many incorrect instances in the repo :disappointed: 

Per ISG it definitely should be hyphenated. I'll make a note of it and if I ever get a chance, I'll do a little sweep through to fix them all. Thanks for letting me know!

Reply:
I can help with the sweep through. I'll open an issue for the OpenShift web console where it is listed as "Command Line Tools" both in the menu and in the section title.

Reply:
Wonderful, ty!

I actually just did a quick sweep and opened this PR: https://github.com/openshift/openshift-docs/pull/91919.

I fixed instances of where "tool" or "interface" comes after it. "command line" on it own would be fine.

I still need to go through and check to make sure they're all okay, but it's a start at least. I can tag you in the PR after I'm done if you don't mind reviewing as well?

Reply:
Of course, I'll be very happy to review it, thank you.